wiki/Contests/Mapping_Olympiad/en.md

```diff
@@ -0,0 +1,83 @@
+| [\#2: osu!taiko](https://osu.ppy.sh/community/contests/65) | ![][flag_FR] [Nofool](https://osu.ppy.sh/users/672430) | ![][flag_US] [Nifty](https://osu.ppy.sh/users/4956097) | ![][flag_ID] [Volta](https://osu.ppy.sh/users/4154071) | [Results](https://osu.ppy.sh/home/news/2019-09-07-osu!taiko-mapping-olympiad-2-results) |
+| [\#3: osu!mania](https://osu.ppy.sh/community/contests/67) | ![][flag_FI] [PianoLuigi](https://osu.ppy.sh/users/9665915) | ![][flag_KR] [Kawawa](https://osu.ppy.sh/users/4647754) | ![][flag_ID] [\_VianK\_](https://osu.ppy.sh/users/6889055) | [Results](https://osu.ppy.sh/home/news/2019-12-27-osu-mapping-olympiad-3-results) |
+| [\#4: osu!catch](https://osu.ppy.sh/community/contests/69) | ![][flag_SG] [Sinnoh](https://osu.ppy.sh/users/4236057) | ![][flag_KR] [Minato Yukina](https://osu.ppy.sh/users/4647754) | ![][flag_US] [Ascendance](https://osu.ppy.sh/users/2931883) | [Results](https://osu.ppy.sh/home/news/2019-02-06-osu-mapping-olympiad-4-results) |
+| [\#5: Concept Mapping](https://osu.ppy.sh/community/contests/72) | ![][flag_CN] [Necho](https://osu.ppy.sh/users/4086593),![][flag_CN] [Ryuusei Aika](https://osu.ppy.sh/users/7777875) | ![][flag_BE] [yaspo](https://osu.ppy.sh/users/4945926) | ![][flag_CA] [J1\_](https://osu.ppy.sh/users/5918561), ![][flag_HK] [how2miss](https://osu.ppy.sh/users/4477199) | [Results](https://osu.ppy.sh/home/news/2019-05-22-osu-mapping-olympiad-5-results) |
+| [\#6: osu!catch](https://osu.ppy.sh/community/contests/96) | ![][flag_HK] [alienflybot](https://osu.ppy.sh/users/636114) | ![][flag_SG] [Sinnoh](https://osu.ppy.sh/users/4236057) | ![][flag_CN] [Yumeno Himiko](https://osu.ppy.sh/users/1806962) | [Results](https://osu.ppy.sh/home/news/2020-04-23-osu-mapping-olympiad-6-results) |
+| [\#7: osu!mania](https://osu.ppy.sh/community/contests/108) | ![][flag_US] [Toaph Daddy](https://osu.ppy.sh/users/7616811), ![][flag_KR] [Kawawa](https://osu.ppy.sh/users/4647754) | - | ![][flag_PH] [lenpai](https://osu.ppy.sh/users/5314573), ![][flag_CA] [BringoBrango](https://osu.ppy.sh/users/10274043) | [Results](https://osu.ppy.sh/home/news/2020-12-01-osu-mapping-olympiad-7-concludes) |
+
+## Prizes
+
+The osu! Mapping Olympiad contest has its own rewarding system and works under a point system.
+
+- **Contest winner**: 6 points
+- **Second place**: 4 points
+- **Third place**: 2 points
+
+Achieving a certain amount of points can grant the contestant specific prizes:
+
+- As of the second contest's edition, winning a contest grants the contestant an *Elite Mapper* profile title.  
+- Once a contestant earns 6 points, they get rewarded with a **profile badge**, that can progress up to 5 times on every 6 points the contestant gets.
+- Every top 3 contestants from each contest earns a **supporter tag** of **two**, **three** or **four months** depending of a contestant ranking. 
```

Comment:
for the formatting part, it reads easier if you make the whole phrase bold, as opposed **to** frequently **switching** between **bold** and **the** regular **thickness**
```suggestion
Achieving a certain amount of points can grant the contestant specific prizes:

- As of the second contest's edition, winning grants a contestant an *Elite Mapper* profile title.  
- Once a contestant earns 6 points, they get rewarded with a **profile badge**, that can progress up to 5 times on every 6 points the contestant gets.
- Every top 3 contestants from each contest earn a **supporter tag of two, three, or four months**, depending of a contestant's ranking. 
```